kodilla-testing/src/main/java/com/kodilla/testing/collection/OddNumbersExterminator.java

```diff
@@ -0,0 +1,37 @@
+package com.kodilla.testing.collection;
+
+import java.util.ArrayList;
+
+public class OddNumbersExterminator {
+
+    public OddNumbersExterminator() {
+
+    }
+
+    public ArrayList<Integer> exterminate(ArrayList <Integer> numbers) {
+
+        ArrayList<Integer> evenList = new ArrayList<Integer>();
+
+        for (int number : numbers) {
+            System.out.println(number);
+            if (number % 2 == 0) {
+                    evenList.add(number);
+            }
+        }
+        System.out.println("\n");
+        return evenList;
+    }
+
+   public int checkList(ArrayList <Integer> tempNumbers) {
+
```

Comment:
A co to za metoda?

Reply:
potrzebna do testów- w testach nie mogę robić pętli,  wiec zrobiłem metodę do testowania w klasie... chyba że coś nie rozumiem.

Reply:
Jak to w testach nie możesz robić pętli? Możesz mieć pomocniczą metodę, która sprawdzi czy dla danej listy każdy element jest parzysty :-)

Reply:
Generalnie w kodzie nie powinniśmy pisać metod, które są nam potrzebne tylko do testów - tylko w wyjątkowych sytuacjach i wtedy oznaczamy metody @VisibleForTesting